site/en/blog/autoplay/index.md

```diff
@@ -0,0 +1,290 @@
+
+
+
+
+<figure class="w-figure">
+  {% Columns %}
+
+  {% Column %}
+
+  {% Img src="image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/iEG2wgEqo8Sx40QqfS8E.jpeg", alt="Liam Neeson: I will find you and I will pause you.", width="398", height="374" %}
+
+  {% endColumn %}
+
+  {% Column %}
+
+  {% Img src="image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/pl6fnOXI5RWtyXXsfYPg.jpeg", alt="Sean Bean: One does not simply autoplay videoas.", width="335", height="335" %}
+
+  {% endColumn %}
+
+  {% endColumns %}
+  <figcaption class="w-figcaption">
+    Internet memes tagged "autoplay" found on <a href="https://imgflip.com/i/ngd6c">Imgflip</a> and <a href="https://imgur.com/a/p1ZjC">Imgur</a>.
+  </figcaption>
+</figure>
+
+## New behaviors
+
+As you may have [noticed], web browsers are moving towards stricter autoplay
+policies in order to improve the user experience, minimize incentives to install
+ad blockers, and reduce data consumption on expensive and/or constrained
+networks. These changes are intended to give greater control of playback to
+users and to benefit publishers with legitimate use cases.
+
+Chrome's autoplay policies are simple:
+
+- Muted autoplay is always allowed.
+- Autoplay with sound is allowed if:
+    - The user has interacted with the domain (click, tap, etc.).
+    - On desktop, the user's [Media Engagement Index](#media-engagement-index)
+      threshold has been crossed, meaning the user has previously played video
+      with sound.
+    - The user has [added the site to their home screen] on mobile or
+      [installed the PWA](installed-the-pwa) on desktop.
+- Top frames can [delegate autoplay permission](#iframe-delegation) to their iframes to
+  allow autoplay with sound.
+
+
+### Media Engagement Index (MEI)
+
+The MEI measures an individual's propensity to consume media on a site.
+Chrome's [current approach] is a ratio of visits to significant media playback
+events per origin:
+
+- Consumption of the media (audio/video) must be greater than seven seconds.
+- Audio must be present and unmuted.
+- The tab with the video is active.
+- Size of the video (in px) must be greater than [200x140].
+
+From that, Chrome calculates a media engagement score, which is highest on sites
+where media is played on a regular basis. When it is high enough, media is
+allowed to autoplay on desktop only.
+
+A user's MEI is available at the `about://media-engagement` internal page.
+
+<figure class="w-figure">
+  {% Img src="image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/x0XKHYcLKjmDzjfxQxFg.png", alt="Screenshot of about://media-engagement internal page.", width="800", height="280" %}
+  <figcaption class="w-figcaption">
+    Screenshot of the <code>chrome://media-engagement</code> internal page.
+  </figcaption>
+</figure>
+
+### Developer switches
+
+As a developer, you may want to change Chrome autoplay policy behavior locally
+to test your website for different levels of user engagement.
+
+- You can disable the autoplay policy entirely by using a [command line flag]:
```

Comment:
`[command line flag]` link does not work. 

![image](https://user-images.githubusercontent.com/634478/119481092-9df55100-bd52-11eb-9537-388b172cc6d5.png)

Reply:
Fixed.